justfile

```diff
@@ -0,0 +1,43 @@
+# Define variables
+set dotenv-required
+
+DB_CONTAINER_NAME := "mina-mesh-db"
+
+# Download archive DB dump
+dl network:
+    ./tasks/dl.sh --network {{network}}
+
+# Initialize PostgreSQL database
+pg-init:
+    docker run -d --name {{DB_CONTAINER_NAME}} -p 5432:5432 -v $(pwd)/sql_scripts:/docker-entrypoint-initdb.d -e POSTGRES_PASSWORD=$POSTGRES_PASSWORD -e POSTGRES_USER=$POSTGRES_USER postgres postgres -c log_statement=all -c log_min_duration_statement=0
+
+# Wait for PostgreSQL to be ready
+pg-wait:
+    ./tasks/pg_wait.sh $DATABASE_URL
+
+# Apply optimizations to PostgreSQL
+pg-apply-optimizations:
+    sqlx migrate run --source sql/migrations
```

Comment:
That's a good idea, but I suppose the migrations require certain file-names and structure? I get this:

```
$ just pg-apply-optimizations 
sqlx migrate run --source sql/migrations
error: while resolving migrations: error parsing migration filename "apply_search_tx_optimizations.sql"; expected integer version prefix (e.g. `01_foo.sql`)
error: Recipe `pg-apply-optimizations` failed on line 21 with exit code 1
```

Just to note that the sql scripts are also used by the `mina-mesh search-tx-optimizations` command (so that also would need to be adjusted):
```
search-tx-optimizations  Command to apply or drop search transaction optimizations in the archive database.
```